runtime/gc_modron_standard/StandardAccessBarrier.cpp

```diff
@@ -1041,8 +1041,9 @@ void
 MM_StandardAccessBarrier::preMountContinuation(J9VMThread *vmThread, j9object_t contObject)
 {
 #if defined(OMR_GC_CONCURRENT_SCAVENGER)
-	if (_extensions->isConcurrentScavengerInProgress()) {
-		/* concurrent scavenger in progress */
+	/* preMountContinuation is a callback from mutator thread, check if concurrent scavenger currently is in active */
+	if (vmThread->readBarrierRangeCheckTop != 0) {
```

Comment:
the old check would be ok (since this is mutator thread, and if CS is in progress it must be concurrent phase), but ok, let's do it his way

however:
- put '0 !=' at the front
- fix comment: ... if concurrent scavenger is currently active */